Sources/UI/Banners/Banners.swift

```diff
@@ -135,25 +135,6 @@ public extension Banners {
     ///
     /// - Parameter error: an error.
     func show(error: Error) {
-        var error = error
-        var message = "\(error)"
-        
-        if let anyError = (error as? AnyError)?.error {
-            error = anyError
-        }
-        
-        if let clientError = error as? ClientError {
-            message = clientError.errorDescription ?? message
-            
-            if let internalError = clientError.error {
-                error = internalError
-            }
-        }
-        
-        if let clientErrorResponse = error as? ClientErrorResponse {
-            message = "Error #\(clientErrorResponse.code): \(clientErrorResponse.message)"
-        }
```

Comment:
`error.localizedDescription` called on `AnyError` should fire cascade of `localizedDescription` on nested errors and it should have same outcome